doc/modules/svm.rst

```diff
@@ -619,37 +669,37 @@ The decision function is:
     estimator used is :class:`sklearn.linear_model.Ridge <ridge>` regression,
     the relation between them is given as :math:`C = \frac{1}{alpha}`.
 
-.. TODO multiclass case ?/
+LinearSVC
+---------
 
-This parameters can be accessed through the members ``dual_coef_``
-which holds the product :math:`y_i \alpha_i`, ``support_vectors_`` which
-holds the support vectors, and ``intercept_`` which holds the independent
-term :math:`\rho` :
-
-.. topic:: References:
-
- * `"Automatic Capacity Tuning of Very Large VC-dimension Classifiers"
-   <http://citeseerx.ist.psu.edu/viewdoc/summary?doi=10.1.1.17.7215>`_,
-   I. Guyon, B. Boser, V. Vapnik - Advances in neural information
-   processing 1993.
+The primal problem can be equivalently formulated as
 
+.. math::
 
- * `"Support-vector networks"
-   <https://link.springer.com/article/10.1007%2FBF00994018>`_,
-   C. Cortes, V. Vapnik - Machine Learning, 20, 273-297 (1995).
+    \min_ {w, b} \frac{1}{2} w^T w + C \sum_{i=1}\max(0, y_i (w^T \phi(x_i) + b)),
 
+where we make use of the `hinge loss
+<https://en.wikipedia.org/wiki/Hinge_loss>`_. This is the form that is
+directly optimized by :class:`LinearSVC`, but unlike the dual form, this one
+does not involve inner products between samples, so the infamous kernel trick
```

Comment:
it's not _infamous_, is it? ;) (to me the work has a negative connotation, whereas the kernel trick is awesome)

Reply:
ah totally, I didn't know about the negative connotation. Will change